Added setRoleInfo()

MentallyDisturbedNub opened the pull request:
Created a command to update a group role's information, including rank ID, name and description.

Neztore commented:
Thank you for contributing! I see your build is failing due to linting, could you please run `npm run lint` on your code locally and fix the issues raised?

MentallyDisturbedNub commented:
Done. I also removed my test case because the feature requires ownership of the group, and a similar test case was also removed 
 for the same reason (setGroupDescription).

Neztore reviewed (approved):
Sorry for the delay on this - Thank you for contributing.
